### PR TITLE
created Attendee and ActivityAttendee clasees

### LINKED ISSUE #70: Create Attendee and Activity attendee
**Summary.**
Create The Class Attendee and ActivityAttendee in the activities app

### PR DESCRIPTION
**Pull request by camiloserr**
### Summary
Created Attendee and ActivityAttendee classes

### Test Plan
github actions

### Related Issues

Closes #70

### CODE REVIEW

File: acm_web_site/apps/business/activities/models.py [hunk 1]
```diff
@@ -95,3 +95,16 @@ class Tutorial(models.Model):
...
+    name = models.CharField(max_length=100)
+    email = models.EmailField(unique=True)
+
+    def __str__(self):
+        return '%s - %s' % (self.name, self.email)
+
+
+class ActivityAttendee(models.Model):
+    activity = models.ForeignKey(Activity, on_delete=models.CASCADE)
+    member = models.ForeignKey(Member, on_delete=models.CASCADE)
```
Comment (BeltranDaniel) [inline acm_web_site/apps/business/activities/models.py]: Shouldn't this foreign key refer to the Attendee class?

**Review by BeltranDaniel — CHANGES_REQUESTED** [review]

File: acm_web_site/apps/business/activities/models.py (hunk 1 above)
Comment (camiloserr) [reply acm_web_site/apps/business/activities/models.py]: yes, my bad

**Review by DorianMoreno — APPROVED** [review]
GJ

**Review by BeltranDaniel — APPROVED** [review]
Good work

**Pull request merged**